### PR TITLE
Issue #KN-617 feat: Video Player angular version update from 13 to 14

### PR DESCRIPTION
**Pull request by santoshilimi**
<table>
<td>
  <a href="https://project-sunbird.atlassian.net/browse/KN-617" title="KN-617" target="_blank">RFCKN-617</a>  Video Player angular version update from 12 to 14 with IVY
  </td></table>
  <br />
 

Implementation Video: https://drive.google.com/file/d/1LkDGm2LAlVSqltbzY1iav1xL6T-88jD_/view?usp=sharing

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: build-wc.js [hunk 1]
```diff
@@ -8,12 +8,19 @@ const build = async () => {
     "./dist/video-player-wc/scripts.js",
     "./dist/video-player-wc/vendor.js",
     "./dist/video-player-wc/main.js",
+    "web-component/assets/videojs-markers.js",
```
Comment (HarishGangula) [inline build-wc.js:11]: Before adding they should be minified and uglified to reduce final JS output file size

**Review by HarishGangula — CHANGES_REQUESTED** [review]

File: build-wc.js (hunk 1 above)
Comment (santoshilimi) [reply build-wc.js:11]: Done

Comment (HarishGangula) [reply build-wc.js:11]: Wha is the change in the JS size post this change

Comment (santoshilimi) [reply build-wc.js:11]: Previous js file size: 2.74 MB
Current js file size: 2.72 MB

**Pull request merged**